src/obfuscation/obfuscation_api.h

```diff
@@ -0,0 +1,31 @@
+/*
+* Copyright Redis Ltd. 2016 - present
+ * Licensed under your choice of the Redis Source Available License 2.0 (RSALv2) or
+ * the Server Side Public License v1 (SSPLv1).
+ */
+
+#ifndef OBFUSCATION_API_H
+#define OBFUSCATION_API_H
+#include "redisearch.h"
+
+#define MAX_OBFUSCATED_INDEX_NAME 6/*strlen("Index@")*/ + MAX_UNIQUE_ID_TEXT_LENGTH_UPPER_BOUND + 1/*null terminator*/
```

Comment:
Are these definition only used for tests? Can we move it to the test file?

Reply:
No, in the next obfuscation branches I'm using these macros to create arrays on the stack to house obfuscated names.